hellochain/tutorial/03-types.md

```diff
@@ -10,4 +10,5 @@ define this struct and other necessary components in a types file.
 
 Save this to `x/greeter/internal/types/types.go`
 
-<<< @/hellochain/x/greeter/internal/types/types.go
++++ https://github.com/cosmos/sdk-tutorials/blob/master/hellochain/x/greeter/internal/types/types.go
```

Comment:
hmm relative paths dont work anymore? we have to use URLs? why is that if you dont mind me asking

Reply:
We didn't use relative paths in the SDK docs, so this feature got kind of forgotten. I've added it back and modified MDs accordingly.